src/EnergyPlus/ConvectionCoefficients.hh

```diff
@@ -316,50 +234,30 @@ namespace ConvectionCoefficients {
         BoundingBoxVertStruct XuYdZu; // 7 hi x, low y, hi z
         BoundingBoxVertStruct XuYuZu; // 8 hi x, hi y, hi z
         Array1D<Vector> BoundSurf;    // long edge of roof group bounding surface
-        Real64 Area;
-        Real64 Perimeter;
-        Real64 Height;
+        Real64 Area = 0.0;
+        Real64 Perimeter = 0.0;
+        Real64 Height = 0.0;
 
         // Default Constructor
-        RoofGeoCharacteristicsStruct() : BoundSurf(4), Area(0.0), Perimeter(0.0), Height(0.0)
+        RoofGeoCharacteristicsStruct() : BoundSurf(4)
         {
         }
     };
 
     struct FacadeGeoCharacteristicsStruct
     {
         // Members
-        Real64 AzimuthRangeLow;
-        Real64 AzimuthRangeHi;
-        Real64 Zmax;
-        Real64 Zmin;
-        Real64 Ymax;
-        Real64 Ymin;
-        Real64 Xmax;
-        Real64 Xmin;
-        Real64 Area;
-        Real64 Perimeter;
-        Real64 Height;
-
-        // Default Constructor
-        FacadeGeoCharacteristicsStruct() = default;
-
-        // Member Constructor
-        FacadeGeoCharacteristicsStruct(Real64 const AzimuthRangeLow,
-                                       Real64 const AzimuthRangeHi,
-                                       Real64 const Zmax,
-                                       Real64 const Zmin,
-                                       Real64 const Ymax,
-                                       Real64 const Ymin,
-                                       Real64 const Xmax,
-                                       Real64 const Xmin,
-                                       Real64 const Area,
-                                       Real64 const Perimeter,
-                                       Real64 const Height)
-            : AzimuthRangeLow(AzimuthRangeLow), AzimuthRangeHi(AzimuthRangeHi), Zmax(Zmax), Zmin(Zmin), Ymax(Ymax), Ymin(Ymin), Xmax(Xmax),
-              Xmin(Xmin), Area(Area), Perimeter(Perimeter), Height(Height)
-        {
-        }
+        Real64 AzimuthRangeLow = AzimuthRangeLow;
```

Comment:
Ummmmm ... what's this?

Reply:
OK, so in this case, this struct has a member-constructor, where the caller provides all the inputs as arguments and they are applied to the class members.  We should indeed leave the member constructor here...unless it's not being used anywhere in the code (hint: if it compiled without it then it must not be used anywhere).  We are really just trying to clean up the structs that only use default constructors with the initial values listed in the initializer lists.

Reply:
I wasn't asking about the member constructor, I was asking about what `Real64 AzimuthRangeLow = AzimuthRangeLow;` is intended to accomplish and actually accomplishes. It looks like it doesn't do anything.